### PR TITLE
Update tls cert secret validation due VerifyCertSecrets() change

### PR DESCRIPTION
**Pull request by stuggi**
Depends-On: https://github.com/openstack-k8s-operators/lib-common/pull/559

Jira: [OSPRH-9991](https://issues.redhat.com//browse/OSPRH-9991)

### CODE REVIEW

File: controllers/amphoracontroller_controller.go [hunk 1]
```diff
@@ -344,15 +344,21 @@ func (r *OctaviaAmphoraControllerReconciler) reconcileNormal(ctx context.Context
 			},
 		)
 		if err != nil {
+			if k8s_errors.IsNotFound(err) {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.TLSInputReadyCondition,
+					condition.RequestedReason,
+					condition.SeverityInfo,
+					fmt.Sprintf(condition.TLSInputReadyWaitingMessage, instance.Spec.TLS.CaBundleSecretName)))
+				return ctrl.Result{}, nil
```
Comment (abays) [inline controllers/amphoracontroller_controller.go:353]: Same here regarding `RequeueAfter`

File: controllers/octaviaapi_controller.go [hunk 2]
```diff
@@ -623,33 +623,45 @@ func (r *OctaviaAPIReconciler) reconcileNormal(ctx context.Context, instance *oc
 			},
 		)
 		if err != nil {
+			if k8s_errors.IsNotFound(err) {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.TLSInputReadyCondition,
+					condition.RequestedReason,
+					condition.SeverityInfo,
+					fmt.Sprintf(condition.TLSInputReadyWaitingMessage, instance.Spec.TLS.CaBundleSecretName)))
+				return ctrl.Result{}, nil
```
Comment (abays) [inline controllers/octaviaapi_controller.go:632]: Another `RequeueAfter` needed?

File: controllers/octaviaapi_controller.go [hunk 3]
```diff
@@ -623,33 +623,45 @@ func (r *OctaviaAPIReconciler) reconcileNormal(ctx context.Context, instance *oc
...
-		certsHash, ctrlResult, err := instance.Spec.TLS.API.ValidateCertSecrets(ctx, helper, instance.Namespace)
+		certsHash, err := instance.Spec.TLS.API.ValidateCertSecrets(ctx, helper, instance.Namespace)
 		if err != nil {
+			if k8s_errors.IsNotFound(err) {
+				instance.Status.Conditions.Set(condition.FalseCondition(
+					condition.TLSInputReadyCondition,
+					condition.RequestedReason,
+					condition.SeverityInfo,
+					fmt.Sprintf(condition.TLSInputReadyWaitingMessage, err.Error())))
+				return ctrl.Result{}, nil
```
Comment (abays) [inline controllers/octaviaapi_controller.go:656]: Another `RequeueAfter` needed?

File: controllers/amphoracontroller_controller.go (hunk 1 above)
Comment (stuggi) [reply controllers/amphoracontroller_controller.go:353]: we index/watch the named cert secret, should not be required, or why you think we should have it?

**Pull request merged**